Polyclinic/Polyclinic.Test/Test/PolyclinicTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace Polyclinic.Test.Test
+
+/// <summary>
+/// Unit tests for Polyclinic
+/// </summary>
+{
+    public class PolyclinicTest(DataSeed seed) : IClassFixture<DataSeed>
+    {
+
+        /// <summary>
+        /// Test to fetch doctors with at least 10 years of experience
+        /// </summary>
+        [Fact]
+        public void GetDoctorsWithExperienceAtLeast10Years()
+        {
+            var expectedCount = 6;
+
+            var experiencedDoctors = seed.Doctors.Where(d => d.ExperienceYears >= 10).ToList();
+
+            Assert.NotNull(experiencedDoctors);
+            Assert.All(experiencedDoctors, d => Assert.True(d.ExperienceYears >= 10));
+            Assert.Equal(expectedCount, experiencedDoctors.Count);
+        }
+
+        /// <summary>
+        /// Test to get patients by specific doctor ordered by name
+        /// </summary>
+        [Fact]
+        public void GetPatientsByDoctorOrderedByName()
+        {
+            var doctorId = 7;
+
+            var patientIds = seed.Appointments
+                .Where(a => a.DoctorId == doctorId)
+                .Select(a => a.PatientId)
+                .Distinct();
+
+            var result = seed.Patients
+                .Where(p => patientIds.Contains(p.ID))
+                .OrderBy(p => p.FullName)
+                .ToList();
+
+            Assert.True(result.Count > 0);
+            Assert.Equal(result, result.OrderBy(p => p.FullName).ToList());
```

Comment:
Можно упростить https://learn.microsoft.com/ru-ru/dotnet/fundamentals/code-analysis/style-rules/ide0305.
Тут и по коду ниже. Visual Studio вам об этом подскажет на вкладке Error List.
```suggestion
            Assert.Equal(result, [.. result.OrderBy(p => p.FullName)]);
```